src/arrays/ArrayUtil.java

```diff
@@ -26,8 +42,11 @@ public static void create2dArray(int x, int y) {
      * @return new array.
      */
     public static int[] addFirst(int[] array, int valueToAdd) {
-
-        return null;
+        int[]result = new int[array.length+1];
+        System.arraycopy(array,0,result,1,array.length);
```

Comment:
A working solution, but please rewrite this without using `System.arraycopy`, try to implement using only one array